### ISSUE TITLE
Macros are not visible during semantic validation

### ISSUE THREAD

**Issue by elight** [elight/coulda#1]
The below currently fails.

```
Feature "feature name" do
  in_order_to "foo"
  as_a "bar"
  i_want_to "blech"

  def given_a_macro
    Given "a method that calls Given" do; end
  end

  Scenario "that is live" do
    given_a_macro
    When "bar" do; end
    Then "blech" do; end
  end
end
```

**Comment by elight** [comment]
Fixing for 0.2

**Comment by elight** [comment]
Fixed in 0.2.0
